src/containers/blocks.jsx

```diff
@@ -417,7 +417,7 @@ class Blocks extends React.Component {
                     // Anything else will be picked up from the XML attached to the block instance.
                     const extendedOpcode = `${categoryInfo.id}_${blockInfo.info.opcode}`;
                     const blockDefinition =
-                        defineDynamicBlock(this.ScratchBlocks, categoryInfo, blockInfo, extendedOpcode);
+                        defineDynamicBlock(this, this.ScratchBlocks, categoryInfo, blockInfo, extendedOpcode);
```

Comment:
Need to pass in the whole gui container so that the built in `RENAME_A_VARIABLE` callback (and when we add support for editing custom procedures, the built in `EDIT_A_PROCEDURE` callback) can open the appropriate modal.